Maestro/subscriptions.json

```
@@ -725,6 +725,9 @@
         "https://github.com/dotnet/project-system/blob/master/**/*",
         "https://github.com/dotnet/project-system/blob/release/**/*",
         "https://github.com/dotnet/project-system-tools/blob/master/**/*",
+	"https://github.com/dotnet/roslyn/blob/master/**/*",
```

I think we also need the vs-deps branch
```suggestion
	"https://github.com/dotnet/roslyn/blob/master/**/*",
	"https://github.com/dotnet/roslyn/blob/master-vs-deps/**/*",
```